crates/sui-storage/src/lib.rs

```diff
@@ -19,7 +21,10 @@ pub fn default_db_options(
     cache_capacity: Option<usize>,
 ) -> (DBOptions, DBOptions) {
     let mut db_options = db_options
-        .map(|o| DBOptions { options: o })
+        .map(|o| DBOptions {
+            options: o,
+            rw_options: ReadWriteOptions::default(),
```

Comment:
why do we need this if it's the default value?

Reply:
hmm, not sure what is the rusty way of doing it - i can look around and fix it later